vendor/bat-native-ads/src/bat/ads/internal/account/ad_rewards/ad_rewards.cc

```diff
@@ -283,7 +283,8 @@ void AdRewards::OnGetPayments(const UrlResponse& url_response) {
   }
 
   if (!payments_->SetFromJson(url_response.body)) {
-    BLOG(0, "Failed to parse payment balance: " << url_response.body);
+    BLOG(0, "Failed to parse payment balance");
+    BLOG(1, "Response body: " << url_response.body);
```

Comment:
nit: `Payment balance response body:` as these may not be printed close together